File: lab9/Yen.cs
using System;

class Yen : Currency
{
    public Yen(double amount) : base(amount) { }

    public override double ToUAH() => Amount * 0.25;

    public override void Print()
    {
        Console.WriteLine($"{Amount} Yen = {ToUAH()} UAH");
    }
}

File: lab9/Yen.java
class Yen extends Currency {
    public Yen(double amount) {
        super(amount);
    }

    @Override
    public double toUAH() {
        return amount * 0.25;
    }

    @Override
    public void print() {
        System.out.println(amount + " Yen = " + toUAH() + " UAH");
    }
}
